java-client/src/main/java/org/opensearch/client/opensearch/_types/query_dsl/WrapperQuery.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
The Elasticsearch header could be dropped, this is a new file, right?

Reply:
I suppose - I copied it from the other PR which has this header, so it's actually less clear that I can just drop it depending on where the other PR sourced the file.